tests/test_SequenceManager.cc

```diff
@@ -72,12 +75,14 @@ std::vector<std::string> collect_lua_filenames(const std::filesystem::path& path
 
 TEST_CASE("SequenceManager: Constructor with path", "[SequenceManager]")
 {
+    std::filesystem::remove_all("./another/path/to/sequences");
     SequenceManager sm{"./another/path/to/sequences"};
     REQUIRE(sm.get_path() == "./another/path/to/sequences");
 }
 
 TEST_CASE("SequenceManager: Move constructor", "[SequenceManager]")
 {
+    std::filesystem::remove_all("unit_test_files");
```

Comment:
We have that in the `main()` of our unit test suite already:
```cpp

int main(int argc, char* argv[])
{
    if (std::filesystem::exists("unit_test_files"))
        std::filesystem::remove_all("unit_test_files");

    return Catch::Session().run(argc, argv);
}
```
Isn't that enough?